templates/download.html

```diff
@@ -1,81 +1,26 @@
 {% extends "base.html" %}
 {% load i18n %}
 {% block content %}
-    <h2>{% trans "Download information about units" %}</h2>
-    <form action='{% url 'download' %}' method='POST'>
-        {% csrf_token %}
-        <div class="row">
-            <div class="col-sm-12">
-            {% if form.download_type.errors %}
-                <div class="form-group has-error has-feedback">
-            {% else %}
-                <div class="form-group">
-            {% endif %}
-                    <label class="control-label">{{ form.fields.download_type.label }}</label>
-                    <select class="form-control" name="download_type">
-                        <option value="" selected>----------</option>
-                        {% for download_type, name in form.fields.download_type.choices %}
-                            <option value="{{ download_type }}">{{ name }}</option>
-                        {% endfor %}
-                    </select>
-                    {% if form.download_type.errors %}
-                        {% for error in form.download_type.errors %}
-                        <span class="help-block"><i class="fa fa-remove"> </i> {{error}}</span>
-                        {% endfor %}
-                    {% endif %}
+    <section class="container text-center">
+      <h1>{% trans "Use Our Data." %}</h1>
+      <h1>{% trans "Hold Perpetrators Accountable." %}</h1>
+      <div class="row download-lead-container">
+          <p class="lead download-lead-text">Download the data that powers WhoWasInCommand to answer your own questions about the structure, behaviour and people in charge of security forces like the police and army.</p>
```

Comment:
All text in this template needs to be wrapped in trans tags so it can be translated.

Reply:
Thanks for the heads up.